### PR TITLE
Load local GraphQL schema

### LINKED ISSUE #2743: Add support for importing GraphQL schemas from local files.
Complementary to #2629 ...

Sometimes server deployments disable GraphQL introspection. It would be nice if in those situations Insomnia could load the schema from a local SDL file.

### PR DESCRIPTION
**Pull request by lodi-g**
I often work with GraphQL, and sometimes the introspection requests are blocked. In these case, the developers give me a GraphQL schema so I can still have the documentation. 
However, Insomnia does not support importing local schema files, only fetching from internet. Therefore I must use Postman for this case, but I'd prefer to use Insomnia.

Closes #2743

### CODE REVIEW

**Comment by lodi-g** [comment]
If approved I can also work on #2629 :+1:

**Comment by nijikokun** [comment]
> If approved I can also work on #2629 👍

Really enjoy the tooltip, extremely helpful. Would be glad to see movement on #2629 as well!

File: packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.js [hunk 1]
```diff
@@ -313,6 +315,56 @@ class GraphQLEditor extends React.PureComponent<Props, State> {
...
+      filters: [
+        {
+          name: 'GraphQL file',
+          extensions: ['', 'json'],
+        },
+      ],
+    };
+
+    const { canceled, filePaths: paths } = await electron.remote.dialog.showOpenDialog(options);
+
```
Comment (apisaint) [inline packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.js]: Would suggest to remove newline between assignment and early exit

**Comment by lodi-g** [comment]
Hi @Kong @apisaint , any update on this? 
The package-lock has conflict but it can be generated again anyway.

**Comment by lodi-g** [comment]
See #3803

**Pull request closed without merging**